primitives/avn-common/src/event_types.rs

```diff
@@ -0,0 +1,757 @@
+use codec::{Decode, Encode};
+use hex_literal::hex;
+#[cfg(feature = "std")]
+use serde::{Deserialize, Serialize};
+use sp_core::{H160, H256, H512, U256};
+use sp_runtime::{scale_info::TypeInfo, traits::Member, DispatchResult};
+use sp_std::{convert::TryInto, vec::Vec};
+
+// ================================= Events Types ====================================
+
+const WORD_LENGTH: usize = 32; // basic word type for Ethereum is 32 bytes
+const HALF_WORD_LENGTH: usize = 16; // needed for creating a u128
+const TWENTY_FOUR_BYTES: usize = 24; // needed for creating a u64
+const TWENTY_EIGHT_BYTES: usize = 28; // needed for creating a u32
```

Comment:
This is also a strange comment, for similar reasons. A u32 is 4 bytes.

Reply:
Please see usage of the constant.

Reply:
I did, and that is why I proposed a name change that IMO communicates the usage more effectively.

Reply:
> As discussed before, for as long as the repo is not used to produced the node binary, and its used to host library crates, the cargo.lock shouldn't be checked in https://doc.rust-lang.org/cargo/faq.html#why-do-binaries-have-cargolock-in-version-control-but-not-libraries I would prefer if we removed it and added it to gitignore.

Ok, we will review the setup of the repo and the usage of that file and decide later.